gridstatus/nyiso.py

```diff
@@ -942,6 +942,10 @@ def get_capacity_prices(self, date=None, verbose=False):
             year_code = 27447313
         elif date.year == 2023:
             year_code = 35397361
+        elif date.year == 2024:
+            year_code = 42146126
+        elif date.year == 2025:
+            year_code = 48997190
```

Comment:
@keelerh Thanks for the PR. Could you update the `test_get_capacity_prices` test for these changes? You should be able to copy and paste the statements from previous years.

https://github.com/gridstatus/gridstatus/blob/1ed59cfaf429c62aaa5cf423ff72420afce0f7c7/gridstatus/tests/source_specific/test_nyiso.py#L21-L33

Reply:
@WillKoehrsen tests now updated and passing